src/usr/local/www/diag_packet_capture.php

```diff
@@ -200,8 +200,11 @@ function fixup_host($value, $position) {
 	}
 
 	if ($port != "") {
-		if (!is_port(strip_not($port))) {
-			$input_errors[] = gettext("Invalid value specified for port.");
+		$ports = explode(" ",$port);
```

Comment:
Using space is inconsistent with the other fields on the page. The **Host Address** field supports multiple values and has different characters to denote logic (`,` is `and`, `|` is `or`). This feature should match that. Someone might want to match both source and destination port which would be `and` here.